include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -320,7 +320,8 @@ template <class T> struct zero_sized {
   using size_type = uint8_t;
   static constexpr T *storage_data() noexcept { return nullptr; }
   static constexpr size_type storage_size() noexcept { return 0; }
-  static constexpr void unsafe_set_size(size_t new_size) noexcept {
+  static constexpr void
+  unsafe_set_size([[maybe_unused]] size_t new_size) noexcept {
     IV_EXPECT(new_size == 0 &&
               "tried to change size of empty storage to non-zero value");
   }
```

Comment:
```suggestion
  static constexpr void
  unsafe_set_size(size_t) noexcept {}
```
Does this work?
This is assuming we remove all the assertion related artifact.